README.md

```diff
@@ -1,38 +1,137 @@
+This script is to allow student grades on Spanish placement exams in
+Canvas to be automatically added to MPathways.  It runs
+periodically as a batch script.  It needs little attention unless
+there are external changes to authentication information or data
+formats.  The script is silent except when there are grades processed.
+If there is new grade information the script will email a 
+summary report to an mcommunity group.  The summary report is also
+written to the log every time the script runs.
 
-Script to get SPE grades (after certain date) from Unizin data warehouse and 
-put in MPathways.  Use ESB calls to get the grades and write grades.  Script
-will locally store the time of the last successful request for grades to make
-future requests efficient.  All other information is available via the ESB or 
-is static and can be stored in properties files.
+SPE will keep track of the last time it requested grade data so that
+grades won't constantly be requested or processed.  If the date get
+corrupted it can be adjusted manually.  See SPECIAL PROCESSING below
+for details.
 
-Properties
-- esb connection information
-- query specific information
-- default or override date
-- logging information
-- persist string class configuration 
+# Design
+SPE is a Spring Boot java application. It runs in a continuous loop  but has 
+built in pauses so that it will completely process input periodically. This 
+built in wait should be replaced by explicit cron functionality but the 
+options for that
+are current quite limited in OpenShift.
 
-Design
-- Spring boot batch application which is run as a script on demand.
-- Use ESB to connect to Data warehouse and MPathways.
-- Use WAPI java wrapper to simplify connections to the ESB.
-- Use a custom simple string persistence class to allow 
-storing/reading/writing a single string to preserve a small amount of data
-with very low overhead.
-- Use email for reporting.
-- Expected to run on OpenShift and use ELK for log management.
+The job is nearly stateless.  Grades are retrieved and updated
+through an ESB API.  Non-secure configuration is bundled with the
+application. Secure information is provided through OpenShift
+Secrets.  Summary information is logged and, when there grade
+activity, is sent to the MCommunity group *TBD*.
 
+A very small amount of disk storage is used to record the last request
+date.  This prevents continual re-processing of grades. 
+See SPECIAL FEATURES for details.
+ 
 
-## Production
+# Running SPE
 
-OpenShift
-- needs small persistent file store
-- scheduling for daily run
-- can override the last grade date in properties file or command line value. 
-This allows adjustments when there is an interruption in the scheduled
-running.
+SPE is configured to run on OpenShift.  Projects and running instances
+are provided in the UM instance.
 
+# Configuration / Properties
 
+SPE takes advantage of the Spring profile capability. Profiles are property files are named 
```

Comment:
A little helpful link to the Spring profile capabilities would be a nice to have